OpenRA.Mods.Common/UtilityCommands/OutputActorSequencesCommand.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class OutputActorSequenceCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--actor-sequences"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return args.Length == 2 || args.Length == 3;
+		}
+
+		[Desc("ACTOR-TYPE [PATH/TO/MAP]", "Display the finalized, merged MiniYaml sequence tree for the given actor type. Input values are case-sensitive.")]
```

Comment:
The top level sequence-yaml nodes aren't necessarily actors, so I would avoid using that terminology here.

We have Images at the top level, each of which have N Sequences.